src/client/Client.cc

```diff
@@ -5964,6 +5966,16 @@ void Client::tick()
     }
   }
 
+  {
+    chrono::duration<double, std::milli> ticks_past_since_req =
```

Comment:
I don't know when and how I made the mistake here (this will make client wait in milliseconds); found it while testing locally after replacing `get_latest_version()` by `maybe_request_map()`. I'll fix in the upcoming push.